index.js

```diff
@@ -169,19 +170,25 @@ function verifyStages(stages, jobs) {
 /**
  * Parse the configuration from a screwdriver.yaml
  * @method configParser
- * @param   {Object}               config
- * @param   {String}               config.yaml                Contents of screwdriver.yaml
- * @param   {TemplateFactory}      config.templateFactory     Template Factory to get templates
- * @param   {BuildClusterFactory}  config.buildClusterFactory Build cluster Factory to get build clusters
- * @param   {TriggerFactory}       [config.triggerFactory]    Trigger Factory to find external triggers
- * @param   {Number}               [config.pipelineId]        ID of the current pipeline
- * @param   {Boolean}              [config.notificationsValidationErr]  Throw error when notifications validation fails (default true);
- *                                                                      otherwise return warning
+ * @param   {Object}                          config
+ * @param   {String}                          config.yaml                                 Contents of screwdriver.yaml
+ * @param   {TemplateFactory}                 config.templateFactory                      Template Factory to get templates
+ * @param   {PipelineTemplateVersionFactory}  config.pipelineTemplateVersionFactory       PiplineTemplateVersion Factory to get pipeline templates
```

Comment:
PipelineTemplateVersion

Reply:
what does that suppose to mean?

Reply:
you have a typo https://github.com/screwdriver-cd/config-parser/pull/149/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R176